src/bundles/Elsa.Server.Web/Program.cs

```diff
@@ -26,8 +26,8 @@
 const bool useProtoActor = false;
 const bool useHangfire = false;
 const bool useQuartz = true;
-const bool useMassTransit = true;
-const bool useMassTransitAzureServiceBus = false;
+const bool useMassTransit = false;
+const bool useMassTransitAzureServiceBus = true;
```

Comment:
This will do nothing since the useMassTransit is already set to false.

Reply:
You're correct.